integration/integration_test.go

```diff
@@ -893,7 +902,7 @@ func meetsRequirements() bool {
 func containerDiff(t *testing.T, image1, image2 string, flags ...string) []byte {
 	flags = append([]string{"diff"}, flags...)
 	flags = append(flags, image1, image2,
-		"-q", "--type=file", "--type=metadata", "--json")
+		"-q", "--type=file", "--type=metadata", "--history", "--json")
```

Comment:
I guess neither of `--type=file` and `--type=metadata` will capture the setcap situation. `--history` should be able to because of the number of image layers is different if setcap change is discarded.